rnote-cli/src/cli.rs

```diff
@@ -159,8 +190,8 @@ pub(crate) fn create_doc_export_prefs_from_args(
 
 pub(crate) async fn export_to_file(
     engine: &mut RnoteEngine,
-    rnote_file: PathBuf,
-    output_file: PathBuf,
+    rnote_file: &PathBuf,
+    output_file: &PathBuf,
```

Comment:
a little pedantic, but `impl AsRef<Path>` would be better

Reply:
im fairly new to rust and i didn't really understand the point of the `impl AsRef<Path>`

Reply:
with it you can either give it an allocated `PathBuf`, `&PathBuf` or `&Path`. More important in libraries that want to provide flexible API but it is useful here too, for the future